packages/webdriver/src/utils.js

```diff
@@ -309,9 +309,12 @@ export function getErrorFromResponseBody (body) {
         return new Error('unknown error')
     }
 
-    return new Error(
-        body.value.message ||
-        body.value.class ||
-        'unknown error'
-    )
+    class CustomError extends Error {
```

Comment:
let's but this outside of `getErrorFromResponseBody` export it and have a unit test for it 😉

Reply:
Done.